### PR TITLE
Place filters above charts

### LINKED ISSUE #251: Display Filters above chart

### PR DESCRIPTION
**Pull request by Elkrival**
There was a discussion on how to present the filters on the charts page.

https://gnardog.slack.com/archives/C03KGHSSTNC/p1673621197886439

Screen Shot 2023-01-17 at 10 03 46 AM

Screen Shot 2023-01-17 at 10 16 06 AM

### CODE REVIEW

File: views/ViewChart.react.js [hunk 1]
```diff
@@ -28,24 +24,15 @@ const ViewChart = ({ graph, classes }) => {
...
-            className={classes.showFilterButton}
-            onClick={() => openFilterMenu()}
-          >
-            <FilterListIcon className={classes.leftIcon} />
-            Filters
-          </Button>
         </div>
       )}
-      <Drawer anchor="left" open={showFilterMenu} onClose={onCloseFilterMenu}>
+      <div style={{ paddingBottom: '25px' }}>
```
Comment (taylorkearns) [inline views/ViewChart.react.js]: Should this be in the CSS file?

Comment (Elkrival) [reply views/ViewChart.react.js]: Thank you for this. 
https://github.com/AMP-SCZ/dpdash/pull/252/commits/e62b6cca672da42cca9aab1f550d26e78d59e51c

**Comment by mikestone14** [comment]
My initial $0.02 - I think we can improve the spacing (maybe make the filters within each row closer together and the vertical spacing between the filter form and the chart great) and change the button size to a more standard size vs block element taking up the full width.

**Comment by Elkrival** [comment]
@mikestone14 Thank you for the design feedback. I've made the changes and this is what it looks like now. 
https://github.com/AMP-SCZ/dpdash/pull/252#:~:text=%E2%80%A6-,15cb0ac,-Merge%20state
Screen Shot 2023-01-19 at 9 29 28 AM

File: views/styles/chart_styles.js [hunk 2]
```diff
@@ -149,31 +149,32 @@ export const chartStyles = (theme) => ({
     justifyContent: 'space-between',
   },
   filterFormContainer: {
-    paddingBottom: '25px',
+    paddingBottom: '75px',
   },
   filters: {
-    width: '100%',
+    width: '35%',
```
Comment (mikestone14) [inline views/styles/chart_styles.js]: IMO we shouldn't use percent widths (other than 100% widths). They just get really wonky on different screen sizes (like an extra wide monitor will display these values super wide). Is there a reason we can't provide a specific width value in pixels?

Comment (Elkrival) [reply views/styles/chart_styles.js]: https://github.com/AMP-SCZ/dpdash/pull/252/commits/6b61bdf3db7dd70698364ca5a69293948c807ef8

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**